jkube-kit/jkube-kit-wildfly-jar/src/main/java/org/eclipse/jkube/wildfly/jar/enricher/WildflyJARHealthCheckEnricher.java

```diff
@@ -134,39 +154,16 @@ private boolean isAvailable() {
         return config.containsKey("cloud");
     }
 
-    protected int getFailureThreshold() {
-        return Configs.asInteger(getConfig(Config.FAILURETHRESHOLD));
-    }
-
-    protected boolean isProbeEnforced() {
-        return Configs.asBoolean(getConfig(Config.ENFORCEPROBES));
-    }
-
-    protected int getSuccessThreshold() {
-        return Configs.asInteger(getConfig(Config.SUCCESSTHRESHOLD));
-    }
-
-    protected String getScheme() {
-        return Configs.asString(getConfig(Config.SCHEME));
-    }
-
     protected int getPort() {
-        return Configs.asInt(getConfig(Config.PORT));
+        return asInt(getConfig(Config.PORT));
     }
 
     protected String getLivenessPath() {
-        return Configs.asString(getConfig(Config.LIVENESSPATH));
-    }
-
-    protected int getLivenessInitialDelay() {
-        return Configs.asInt(getConfig(Config.LIVENESSINITIALDELAY));
+        return Configs.asString(getConfig(Config.LIVENESS_PATH));
     }
 
     protected String getReadinessPath() {
```

Comment:
Not used anymore.

(Tests should be improved to test the actual Enricher behavior)